apex/contrib/csrc/conv_bias_relu/conv_bias_relu.cpp

```diff
@@ -149,8 +149,8 @@ cudnn_frontend::ExecutionPlan& getOrCreatePlan(cudnnHandle_t handle_,
         .setOperationGraph(opGraph)
         .setHeurMode(CUDNN_HEUR_MODE_INSTANT)
         .build();
-      // try 3 times for now as WAR for no heuristic training
-      int max_tries = 3, count = 0;
+      // try 10 times for now as WAR for no heuristic training
+      int max_tries = 10, count = 0;
```

Comment:
Could we flip this around, where instead the number of "successful" engines is capped rather than the number of "failures?" I don't see a reason why trying to limit  the number of failing engines is helpful---we don't do this in upstream code for vanilla cuDNN convs.